docker/pypi/wmagent/bin/manage-common.sh

```diff
@@ -119,7 +177,9 @@ _sql_db_isclean(){
     local wmaDBName=${1:-$wmaDBName}
     case $AGENT_FLAVOR in
         'oracle')
-            echo "Not implemented"
+            local sqlCmd="SELECT COUNT(table_name) FROM user_tables;"
```

Comment:
What happens if you run this command in a clean database (without schema)? Will it consider the db as not cleaned?